docs/guides/states.md

```diff
@@ -248,19 +248,19 @@ const machine = createMachine({
 
 const inactiveState = machine.initialState;
 
-inactiveState.can('TOGGLE'); // true
-inactiveState.can('DO_SOMETHING'); // false
+inactiveState.can({type: 'TOGGLE'}); // true
+inactiveState.can({type: 'DO_SOMETHING'}); // false
 
 // Also takes in full event objects:
 inactiveState.can({
   type: 'DO_SOMETHING',
   data: 42
 }); // false
 
-const activeState = machine.transition(inactiveState, 'TOGGLE');
+const activeState = machine.transition(inactiveState, {type: 'TOGGLE'});
 
-activeState.can('TOGGLE'); // false
-activeState.can('DO_SOMETHING'); // true, since an action will be executed
+activeState.can({type: 'TOGGLE'}); // false
+activeState.can({type: 'DO_SOMETHING'}); // true, since an action will be executed
```

Comment:
```suggestion
activeState.can({ type: 'TOGGLE' }); // false
activeState.can({ type: 'DO_SOMETHING' }); // true, since an action will be executed
```